source/applications/advanced/gamma_correction.py

```diff
@@ -0,0 +1,100 @@
+"""
+Capture a 2D image and apply a configurable gamma correction.
+"""
+
+import argparse
+from pathlib import Path
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    """Configure argparse.
+
+    Returns:
+        argparser arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        description="Capture 2D image and apply gamma correction\
+            \nExample:\n\t $ python gamma_correction.py 2",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    parser.add_argument(
+        "gamma", nargs="?", type=float, help="Gamma correction value", default="2"
```

Comment:
Would it make sense to only accept one argument here instead of using `nargs="?"` ?

Reply:
It was necessary when I wanted to use the default value. When the default value goes away, so does `nargs="?"`